src/app/test-modules/hisb-logger-tester/hisb-logger-tester.module.ts

```diff
@@ -4,6 +4,7 @@ import { LoggerTesterComponent } from './components/logger-tester/logger-tester.
 import { NgxHisbLoggerModule } from 'ngx-hisb-logger';
 import {MatButtonModule} from "@angular/material/button";
 import {MatCardModule} from "@angular/material/card";
+import {LoggerService} from "../../../../projects/ngx-hisb-logger/src/lib/services/logger.service";
```

Comment:
We should not import any of the services across projects like this. Remember that the library code will not have a direct path like the one above. The import should point to the library and should look something like this:

`import {LoggerService} from "ngx-hisb-logger";`